api/accounts/accounts.go

```diff
@@ -31,15 +31,15 @@ type Accounts struct {
 	stater       *state.Stater
 	callGasLimit uint64
 	forkConfig   thor.ForkConfig
-	bft          bft.Finalizer
+	bft          bft.CommitLevel
 }
 
 func New(
 	repo *chain.Repository,
 	stater *state.Stater,
 	callGasLimit uint64,
 	forkConfig thor.ForkConfig,
-	bft bft.Finalizer,
```

Comment:
For the name convention, I would consider verb + `er` suffix is better, wdyt? Can you come up better options?

Reply:
How does `BlockCommitter` sound to you?

Reply:
Do you think `bft.Committer` sounds good?